src/pypromice/process/L1toL2.py

```diff
@@ -141,6 +156,248 @@ def toL2(L1, T_0=273.15, ews=1013.246, ei0=6.1071, eps_overcast=1.,
                                              T_0, T_100, ews, ei0)                   
     return ds
 
+def differenceQC(ds):
```

Comment:
It would make sense to add such functions to a separate module like `pypromice.qc`

Reply:
Yes, it might make sense to take this out of `L1toL2.py` and have all qc-related functions in the `qc` directory? Not sure if there are any implications of moving this other than organizational, but worth considering.

Also, it would be great to add a short description at the top of the docstrings for both `differenceQC` and `percentileQC` that gives a basic description of what the function does.

Reply:
It is definitely something we will do in the future, probably in a future update. A better described docsrting is also a good idea